_episodes/04-pipefilter.md

```diff
@@ -802,27 +802,35 @@ so this matches all the valid data files she has.
 > this.
 >
 > 1.  Can you match the same set of files with basic wildcard expressions
->     that do not use the `[]` syntax? *Hint*: You may need more than one
+>     that do not use the `[]` syntax? *Hint*: You can use more than one
 >     expression.
 >
-> 2.  The expression that you found and the expression from the lesson match the
+> 2.  The expressions that you found and the expression from the lesson match the
 >     same set of files in this example. What is the small difference between the
 >     outputs?
```

Comment:
As you mentioned in #615, this question is clearly written with the "two `ls` commands" solution in mind, but for someone who comes up with the single `ls` solution, this question has no answer. I'd suggest relaxing the wording ("is there any difference" rather than "what is the difference"), or just removing this question, and mentioning the difference in the solution section.